astropy/nddata/utils.py

```diff
@@ -456,3 +469,322 @@ def block_replicate(data, block_size, conserve_sum=True):
         data = data / float(np.prod(block_size))
 
     return data
+
+
+class Cutout2D(object):
+    def __init__(self, data, position, shape, wcs=None, mode='trim',
+                 fill_value=np.nan, copy=False):
+        """
```

Comment:
They work fine in ``__init__``.  I thought it could be either/or and I prefer have the parameters in ``__init__``.  I moved the one-line docstring to the class level (but it also worked fine in `__init__`).  I can also move the description up to class level if you want.